app/views/static_pages/index.html.erb

```diff
@@ -1,2 +1,109 @@
-<h1>Home Page</h1>
+<html>
+  <body>
+    <br />
```

Comment:
I'll leave them here for now- not sure how I would do CSS spacing around a booyah-box. Do I add another part to the class?